### PR TITLE
Add Inbox.lv

### PR DESCRIPTION
**Pull request by Yang-Nankai**
Inbox.lv is a Latvian-based email service provider offering free email accounts with advanced features such as spam filtering, cloud storage, and a calendar. It also provides tools for managing contacts, sending attachments, and organizing emails efficiently. Inbox.lv offers 2FA for the account login verification process. The website is ranked 2,554 on [Similarweb](https://pro.similarweb.com/#/digitalsuite/websiteanalysis/overview/website-performance/*/999/1m?webSource=Total&key=inbox.lv). The website provides documentation on how to turn on 2FA.

### CODE REVIEW

**Review by hkamran80 — COMMENTED** [review]
Do you know if their regional sites use the same accounts?

https://company.inbox.lv/localization/

**Comment by Yang-Nankai** [comment]
I tested it and found that websites in different regions do not seem to be universal. Below are the different situations when I log in under inbox.vg and inbox.lv. Under inbox.vg, it shows that my account or password is wrong, but under inbox.lv, I can successfully reach the verification steps. @hkamran80 

**inbox.vg**
image

**inbox.lv**
image

File: img/i/inbox.lv.svg [hunk 1]
```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 48 48"><path fill="#ce2029" d="M24 7a30.4 30.4 0 0 1 20 7v20a30.1 30.1 0 0 1-20 7 30 30 0 0 1-20-7V14a30.5 30.5 0 0 1 20-7m0-4a34.6 34.6 0 0 0-22.6 8A3.5 3.5 0 0 0 0 14v20a3.7 3.7 0 0 0 1.4 3A34.2 34.2 0 0 0 24 45a34.3 34.3 0 0 0 22.6-8 3.7 3.7 0 0 0 1.4-3V14a3.5 3.5 0 0 0-1.4-3A34.6 34.6 0 0 0 24 3Z"/><path fill="#ce2029" d="M12.3 12.9a1.4 1.4 0 0 1 1.6 1.5 1.4 1.4 0 0 1-1.6 1.6 1.4 1.4 0 0 1-1.5-1.6 1.4 1.4 0 0 1 1.5-1.5Zm0 20.6a1.4 1.4 0 0 1-1.5-1.5V18.8a1.4 1.4 0 0 1 1.5-1.6 1.4 1.4 0 0 1 1.6 1.6V32a1.4 1.4 0 0 1-1.6 1.5ZM37 14.4v15.2a1.5 1.5 0 0 1-.6 1.2 14.3 14.3 0 0 1-9.4 3.3 14.3 14.3 0 0 1-9.4-3.3 1.5 1.5 0 0 1-.6-1.2v-8.4a1.5 1.5 0 0 1 .6-1.2 14.5 14.5 0 0 1 9.4-3.4 14.7 14.7 0 0 1 6.9 1.7v-3.9a1.4 1.4 0 0 1 1.5-1.5 1.4 1.4 0 0 1 1.6 1.5ZM27 31a11.6 11.6 0 0 0 6.9-2.2V22a11.8 11.8 0 0 0-6.9-2.2 11.5 11.5 0 0 0-6.9 2.2v6.9A11.3 11.3 0 0 0 27 31Z"/></svg>
```
Comment (fletcherdodd) [inline img/i/inbox.lv.svg]:
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 99.1 86.6"><path fill="#f9f9f9" d="M49.5 6.7c-16.3-.3-32 5.1-44.6 15.2v43A68.6 68.6 0 0 0 49.5 80c16.3.3 32-5.1 44.5-15.3v-43A69.3 69.3 0 0 0 49.5 6.8z"/><path fill="#cc0717" d="M96 16.7A71.3 71.3 0 0 0 49.4 0 71.3 71.3 0 0 0 2.8 16.7 7.3 7.3 0 0 0 0 22.5v41.6a7.6 7.6 0 0 0 3 5.8 70.6 70.6 0 0 0 46.6 16.7 70.5 70.5 0 0 0 46.6-16.7 7.6 7.6 0 0 0 2.9-6V22.5a7.3 7.3 0 0 0-3-5.8zm-5.7 46.9A61.5 61.5 0 0 1 49.5 78 61.6 61.6 0 0 1 8.6 63.6V23A62.4 62.4 0 0 1 49.5 8.6 62.3 62.3 0 0 1 90.3 23zm-6-34.9a2.2 2.2 0 0 0 .8-3 2.3 2.3 0 0 0-3-.8L49.5 43.7 17 25a2.2 2.2 0 0 0-3 .8 2.2 2.2 0 0 0 .8 3L40 43.3 14.7 58a2.2 2.2 0 0 0-.8 3 2.2 2.2 0 0 0 3 .8l27.6-16 5 3 5-3 27.6 16a2.2 2.2 0 0 0 3-.9 2.2 2.2 0 0 0-.8-3L59 43.4z"/></svg>
```
Is this a more suitable logo?

**Comment by Yang-Nankai** [comment]
The logo is much more pleasing to the eye now, Thank you very much!😄 @Carlgo11

**Review by Carlgo11 — APPROVED** [review]

**Pull request merged**